pages/components.js

```diff
@@ -0,0 +1,34 @@
+import { motion } from 'framer-motion'
```

Comment:
I'm having lot of fun with framer motion. Colby Fayock and Will Johnson have some fun tutorials on this that I'm messing around with.